### ISSUE TITLE
Rich Editor - Heading classes/elements

### ISSUE THREAD

**Issue by slafleche** [vanilla/vanilla#6649]
In the **editor**, title always is the `h1` tag and the sub title is always `h2`. If the post is a comment, it will get converted to `h2` and `h3`. The reason for this is for accessibility. The screen reader will read back the element added in the editor, and we want it to match the option we have in the menu (h1 and h2). 

Classes for new discussion:
The title (h1) gets "pageTitle"
The sub title (h2) gets "pageSubTitle"

Classes for comment:
The title (h1 in edit mode, h2 in the page) gets "pageSubTitle"
The sub title (h2 in edit mode, h3 in the page) gets "pageSubSubTitle" 

In other words, the classes **stay the same**, but the _element_ changes depending if we're in edit mode or not.

**Comment by Charrondev** [comment]
We'll need to update the tests to allow for using a different expected output from the server than the frontend.
